src/components/Calculator.js

```diff
@@ -1,82 +1,24 @@
 /* eslint-disable react/forbid-prop-types */
```

Comment:
- [ ] Let's don't disable linter rules. These rules are here to help us write besr code.